src/components/NglViewer/NglViewer.css

```diff
@@ -0,0 +1,20 @@
+.viewport {
+  width: 400px;
+  height: 400px;
+}
+
+.dl {
+  position: absolute;
```

Comment:
The download link and close button are styled so the structure is shown underneath.
 
![localhost_5173_](https://github.com/i-VRESSE/haddock3-analysis-components/assets/1713488/2f0fb688-ebdb-4749-b83f-fc71207e06c6)

Its nice that it maximises the viewport, but can make the file name hard to read.

Reply:
I dont see the same problem, it seems the link and the view are separate :thinking: 
![image](https://github.com/i-VRESSE/haddock3-analysis-components/assets/55081872/e8ffed1f-77a2-416c-8d1f-8f5dd490101a)

Reply:
the problem occurs in the local example html file
![image](https://github.com/i-VRESSE/haddock3-analysis-components/assets/55081872/d4da6ae2-9272-4114-a295-adb2a6780f71)

Reply:
changing the positions in css file should fix it.